### PR TITLE
Feat/substrate v1.0.0 test 2

### PR DESCRIPTION
**Pull request by JasonTulp**
This PR addresses some of the comments left on the initial [Test PR](https://github.com/futureversecom/trn-seed/pull/836).
These include:
 - Using storage_alias for ethy pallet mock storage instead of migration Value and Map
 - Fixing futurepass tests to properly account for Existential Deposit
 - Adding benchmark to Vtx Distribution pallet

### CODE REVIEW

File: pallet/futurepass/src/tests.rs [hunk 1]
```diff
@@ -15,17 +15,19 @@
...
 
 type MockCall = crate::mock::RuntimeCall;
 
-// ProxyDepositBase + ProxyDepositFactor * 1(num of delegates) + 1 for Existential deposit
+// ProxyDepositBase + ProxyDepositFactor * 1(num of delegates)
+// + 1 for Existential deposit of funder
 const FP_CREATION_RESERVE: Balance = 148 + 126 + 1;
-const FP_DELEGATE_RESERVE: Balance = 126 * 1 + 1; // ProxyDepositFactor * 1(num of delegates)
+// ProxyDepositFactor * 1(num of delegates) + Existential Deposit
+const FP_DELEGATE_RESERVE: Balance = 126 * 1 + 1;
```
Comment (surangap) [inline pallet/futurepass/src/tests.rs:30]:
```suggestion
const FP_DELEGATE_RESERVE: Balance = 126 * 1 ;
```
Delegate reserve should not include the ED. By the time the delegates are added, FP should have been created already.

Comment (surangap) [reply pallet/futurepass/src/tests.rs:30]: Might need to review again how we redirect balance calls via AssetExt. you can leave this to me.

**Comment by JasonTulp** [comment]
check coverage

File: pallet/futurepass/src/tests.rs (hunk 1 above)
Comment (surangap) [reply pallet/futurepass/src/tests.rs]: [da66f0a](https://github.com/futureversecom/trn-seed/pull/844/commits/da66f0a426e5952863ae26abca7475ffd19fec8c)

**Review by surangap — APPROVED** [review]
lgtm

**Pull request merged**